front/README.md

```diff
@@ -0,0 +1,28 @@
+## Getting Started
```

Comment:
There will be a readme for each part of build (front/server/builder) and the front readme will have a link to each project otherwise the root readme wiil be too long

Reply:
Do it now please, otherwise it's a bit of a mess :/

Reply:
I made a root README with links to other readme